Duplex steam engines - Booster Engine addition (#899)
openrails/openrails: Source/Documentation/Manual/physics.rst

typically combined to place limits on the power of a locomotive depending
upon the design factors used.

Steam Locomotives with Multiple Engines
.......................................

Some steam locomotives can have multiple steam engines (ie separate steam 
cylinders connected to different wheels), such as the 4-4-4-4 locomotive or 

> an articulated Garrat locomotive.

Reviewer: ```suggestion
an articulated Garratt locomotive.
```

Author: Revised as suggested

> To configure these types of locomotives configurations for multiple steam 

Reviewer: ```suggestion
To configure these types of locomotives, configurations for multiple steam 
```

Author: Revised as suggested

Author: Revised as suggested

> engines need to be added to the engine section of the ENG file, would have the 

Reviewer: ```suggestion
engines need to be added to the engine section of the ENG file. These should have the 
```

Author: Revised as suggested

following format::

    ORTSSteamEngines ( x
        Wheelset (
           
        )
    )

where x = number of steam engines fitted to locomotive.

The following parameters can be used to configure the steam engine::

``NumCylinders`` - number of steam cylinders in engine.
``CylinderStroke`` - stroke of steam cylinder.
``CylinderDiameter`` - diameter of steam cylinder.
``MaxIndicatedHorsepower`` - maximum indicated horsepower of steam engine.
``AttachedAxle`` - the axle wheelset that the steam engine is attached to.

> To specify the engine as a Booster engine the following additional parameters 

Reviewer: ```suggestion
To specify the engine as a Booster engine, the following additional parameters 
```

Author: Revised as suggested
